docs/0.19/basic-usage.md

```diff
@@ -11,6 +11,8 @@ The `CommonMarkConverter` class provides a simple wrapper for converting CommonM
 ~~~php
 <?php
 
+require __DIR__ . '/vendor/autoload.php';
```

Comment:
require_once is what we probably want here?

Reply:
It's funny you should say that, because that's what I wrote at first as well!

But I noticed composer itself uses just `require` (see https://getcomposer.org/doc/01-basic-usage.md#autoloading), so I decided to follow their convention.

Happy to change it though.

Reply:
While I personally prefer `require_once`, I like the idea of following Composer's convention in our examples so beginners can follow along with similar instructions/examples in both places.